fix commit message doc (#251)
Alfresco/alfresco-build-tools: docs/README.md

      - uses: Alfresco/alfresco-build-tools/.github/actions/get-commit-message@ref
```

> To use the action in a proper way, be sure to use a checkout with fetch-depth option as follow:

Reviewer: ```suggestion
This action requires a checkout with fetch-depth option as follow:
```

I would merge the two snippet in a single one (beware of missing dash in the checkout step)
